<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gongn</class>
 <widget class="QWidget" name="gongn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>557</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>734</width>
    <height>557</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>734</width>
    <height>557</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QPushButton" name="XTpB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>系统管理功能</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SJpB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>数据分析功能</string>
   </property>
  </widget>
  <widget class="QPushButton" name="YCpB">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>预测功能</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;功能主界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
